sdk/storage/azblob/zt_test.go

```diff
@@ -533,7 +533,8 @@ func blockIDIntToBase64(blockID int) string {
 func validateStorageError(_assert *assert.Assertions, err error, code StorageErrorCode) {
 	_assert.NotNil(err)
 	var storageError *StorageError
-	_assert.Equal(errors.As(err, &storageError), true)
+	// TOOD: this should really be require.Equal so that if it fails we don't try the next line which will panic
```

Comment:
Just a note, I change these to `require` (which you're correct we should be using over `assert`) in my PR to move to the new test proxy